### ISSUE TITLE
Align currency to center

### ISSUE THREAD

**Issue by bebnev-pro** [9brada6/backbone-js#1]

**Comment by 9brada6** [comment]
Yeah, this was more for functionality than for aspect... In a real world example, probably it would look like this:

![image](https://user-images.githubusercontent.com/19844284/124381379-d03b8c00-dcca-11eb-8c27-d0da83e3fa5c.png)
